src/mon/MgrStatMonitor.cc

```diff
@@ -136,6 +178,7 @@ version_t MgrStatMonitor::get_trim_to()
 
 void MgrStatMonitor::on_active()
 {
+  update_logger();
```

Comment:
Well, doh. Commented on a thread about a former state of the PR, apparently. Please disregard.